### ISSUE TITLE
Please support  IsDelayedDetailViewDataLoadingEnabled in WinDashboardModule

### ISSUE THREAD

**Issue by vimarx** [eXpandFramework/eXpand#160]
After enabling IsDelayedDetailViewDataLoadingEnabled in Xtradashboardtester, it fails when trying to open Dashboard Detailview

	Type:       InvalidOperationException
	Message:    Exception occurs while assigning the 'XpandDetailView, ID:DashboardDefinition_DetailView' view to WinWindow:
Referencia a objeto no establecida como instancia de un objeto.
	Data:       0 entries
	Stack trace:

   en DevExpress.ExpressApp.Frame.SetView(View view, Boolean updateControllers, Frame sourceFrame, Boolean disposeOldView)
   en DevExpress.ExpressApp.Win.WinShowViewStrategyBase.CreateWindow(ShowViewParameters parameters, ShowViewSource showViewSource, Boolean isMain)
   en DevExpress.ExpressApp.Win.WinShowViewStrategyBase.ShowViewInInspector(ShowViewParameters parameters, ShowViewSource showViewSource)
   en DevExpress.ExpressApp.Win.WinShowViewStrategyBase.ShowViewInNewWindow(ShowViewParameters parameters, ShowViewSource showViewSource)
   en DevExpress.ExpressApp.Win.ShowInMultipleWindowsStrategy.ShowViewInNewWindow(ShowViewParameters parameters, ShowViewSource showViewSource)
   en DevExpress.ExpressApp.Frame.ProcessActionCreatedView(ActionBaseEventArgs e)
   en DevExpress.ExpressApp.Actions.ActionBase.OnProcessCreatedView(ActionBaseEventArgs e)
   en DevExpress.ExpressApp.Actions.ActionBase.ExecuteCore(Delegate handler, ActionBaseEventArgs eventArgs)
	----------------
	InnerException:

			Type:       NullReferenceException
			Message:    Referencia a objeto no establecida como instancia de un objeto.
			Data:       0 entries
			Stack trace:

   en Xpand.ExpressApp.Dashboard.Services.Extensions.ToDashboard(IDashboardDefinition dashboardDefinition)
   en Xpand.ExpressApp.Dashboard.Controllers.TestDashboradAssemblyBindingsController.TestDashboradAssemblyBindings()
   en DevExpress.ExpressApp.Utils.BoolList.OnResultValueChanged(Boolean oldResultValue, Boolean resultValue)
   en DevExpress.ExpressApp.Utils.BoolList.EndUpdate()
   en DevExpress.ExpressApp.ViewController.SetView(View newView)
   en DevExpress.ExpressApp.Frame.ActivateViewControllers()
   en DevExpress.ExpressApp.Frame.SetView(View view, Boolean updateControllers, Frame sourceFrame, Boolean disposeOldView)
			InnerException is null

`public XtraDashboardTesterWindowsFormsApplication(){
            InitializeComponent();
            DelayedViewItemsInitialization = true;
            **IsDelayedDetailViewDataLoadingEnabled = true;**
            LastLogonParametersReading += OnLastLogonParametersReading;
        }
`

**Comment by david0villa** [comment]
Yes, I am having the same issue to all dashboard using xpand. Switch off that option then all work

**Comment by apobekiaris** [comment]
i fixed in 18.1.6.2 until then please use this controller in your winmodules.

```cs
using System;
using System.Linq;
using System.Reflection;
using System.Text.RegularExpressions;
using Xpand.ExpressApp.Dashboard.BusinessObjects;
using Xpand.ExpressApp.Dashboard.Services;

namespace XtraDashboardTester.Module.Win.Controllers{
    public class TestDashboradAssemblyBindingsController:Xpand.ExpressApp.Dashboard.Controllers.TestDashboradAssemblyBindingsController {
        protected override void OnActivated() {
            View.CurrentObjectChanged+=ViewOnCurrentObjectChanged;
            TestDashboradAssemblyBindings();
        }

        protected override void OnDeactivated() {
            View.CurrentObjectChanged-=ViewOnCurrentObjectChanged;
        }

        private void TestDashboradAssemblyBindings(){
            var dashboardDefinition = ((IDashboardDefinition) View.CurrentObject);
            AppDomain.CurrentDomain.AssemblyResolve += CurrentDomainOnAssemblyResolve;
            dashboardDefinition?.ToDashboard();
            AppDomain.CurrentDomain.AssemblyResolve -= CurrentDomainOnAssemblyResolve;
        }

        private Assembly CurrentDomainOnAssemblyResolve(object sender, ResolveEventArgs args) {
            var value = Regex.Match(args.Name, @"\A.[^,]*").Value;
            return AppDomain.CurrentDomain.GetAssemblies().FirstOrDefault(assembly => assembly.GetName().Name == value);
        }

        private void ViewOnCurrentObjectChanged(object sender, EventArgs e) {
            TestDashboradAssemblyBindings();
        }
    }
}
```

**Comment by apobekiaris** [comment]
Closing issue for age. Feel free to reopen it at any time.

.Thank you for your contribution.
